client/webserver/site/src/js/markets.js

```diff
@@ -310,6 +310,12 @@ export default class MarketsPage extends BasePage {
     return true
   }
 
+  /* clearPasswordField clears the user password field of the order verification form */
+  clearPasswordField () {
+    const page = this.page
+    page.vPass.value = ''
+  }
+
```

Comment:
No need to write a function if this is a one-liner.

`this.page.vPass.value = ''`

Reply:
That change has been made.